### DISCUSSION TITLE
optical depth

### DISCUSSION THREAD

**Discussion by sderoode** [ARM-Development/comble-mip#30] [General]
The optical depth is asked as an output quantity. In our model it is computed in the solar part of the radiative transfer scheme which we actually don't use as we exclude effects of solar radiation. In any case, it can be computed, but the outcome will depend on some assumptions, for example what to use for the cloud droplet effective radius. In DALES we determine r_eff from the mean volume radius and an assumed cloud droplet size distribution. Is there a protocol for diagnosing the optical depth for COMBLE?

**Comment by fridlind** [comment]
Yes, a computed optical depth also defines the requested cloud_area_fraction, which includes ice hydrometeors, as well as the 2D atmosphere_optical_thickness and pseudo-albedo diagnostics. In DHARMA, we compute this as twice the total projected area of all hydrometeors. We have clarified the [output variables](https://docs.google.com/spreadsheets/d/1Vl8jYGviet7EtXZuQiitrx4NSkV1x27aJAhxxjBb9zI/edit?pli=1#gid=0) notes to clarify that both cloud_area_fraction and optical_depth* quantities requested are "mid-visible" and "may assume geometric scatterers". If another method is used, please note that in the reported variable attributes field.

**Reply by AlejandroBaroPerez** [reply]
Hi @fridlind, with "mid-visible" do you mean 550nm, or some specific range? MIMICA takes the whole range from 200-689 nm to calculate the optical depths in the visible spectrum.

**Comment by fridlind** [comment]
Yes, by mid-visible we mean 550-nm or thereabouts. We do not require a specific method of calculating optical depth in case your code has an in-line method that is already precise, which it sounds like MIMICA does. For spheres when the the ratio of circumference to wavelength exceeds 10, the ratio of total scattering to cross-sectional area approaches 2. This is referred to as the geometric regime. We suggest assuming that liquid and total optical depth can be adequately approximated by assuming that all hydrometeors are in the geometric regime. I would recommend to supply your in-line optical depth and note the wavelength range and method (Mie code?) in the reported variable attributes field. Thank you for the attention to detail!
